mne/preprocessing/maxwell.py

```diff
@@ -449,6 +449,7 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
         raw_sss._data[pos_picks, start:stop] = out_pos_data
 
     # Update info
+    info['dev_head_t'] = recon_trans  # set the reconstruction transform
```

Comment:
`raw_sss` object.

Because it's not actually part of the `sss` bits of the `info` structure (e.g., number of components, etc.). At least that's how I was thinking of it.